ensime-util.el

```diff
@@ -329,6 +329,11 @@ This idiom is preferred over `lexical-let'."
        (ensime-externalize-offset (max (mark) (point))))
     (list (ensime-computed-point) (ensime-computed-point))))
 
+(defun ensime-point-at-offset (offset)
+  (save-excursion
+    (goto-char (ensime-internalize-offset offset))
```

Comment:
What's the difference between ensime-internalize-offset and ensime-point-at-offset?